Yoast/wordpress-seo
# Implement frontend cache

DON'T MERGE YET.

## Summary

This PR can be summarized in the following changelog entry:

* Introduces a cache of Yoast SEO metadata when using object caching, to reduce load on consecutive page loads.

## Relevant technical choices:

* It's... Surprisingly straightforward.
* Closing comment below Yoast SEO metadata says `(cached)` in it.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have a VVV or other test environment with Memcache in active use.
* Check a series of URLs. 
* Check the series of URLs again, see their SEO output is now cached.

## Quality assurance
